src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  display: flex;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  position: absolute;
```

Comment:
why do you need `absolute` here?

Reply:
`padding: 20 0; `will be enough to center links and make header of right hight

Reply:
to position my nav
padding: 20 0; is not working